kafka-streams-framework/src/main/java/org/hypertrace/core/kafkastreams/framework/KafkaStreamsApp.java

```diff
@@ -63,13 +64,16 @@ public abstract class KafkaStreamsApp extends PlatformService {
   public static final String PRE_CREATE_TOPICS = "precreate.topics";
   public static final String KAFKA_STREAMS_CONFIG_KEY = "kafka.streams.config";
   private static final String SHUTDOWN_DURATION = "shutdown.duration";
+  private static final String STARTUP_DELAY = "startup.delay";
   private static final Logger logger = LoggerFactory.getLogger(KafkaStreamsApp.class);
 
   private final GrpcChannelRegistry grpcChannelRegistry;
 
   protected KafkaStreams app;
   private KafkaStreamsMetrics metrics;
   private Duration shutdownDuration;
+  private Duration initialDelay;
```

Comment:
```suggestion
  private Duration initialStartDelay;
```

Reply:
`initialStartupDelay` - both initial and startup together doesn't look correct. I had named it as `startupDelay`, however changed to `initialDelay` based on feedback.